src/OrleansServiceBus/Providers/Streams/EventHub/EventHubAdapterFactory.cs

```diff
@@ -143,7 +143,27 @@ public virtual void Init(IProviderConfiguration providerCfg, string providerName
             {
                 var bufferPool = new FixedSizeObjectPool<FixedSizeBuffer>(adapterSettings.CacheSizeMb, () => new FixedSizeBuffer(1 << 20));
                 var timePurge = new TimePurgePredicate(adapterSettings.DataMinTimeInCache, adapterSettings.DataMaxAgeInCache);
-                CacheFactory = (partition,checkpointer,cacheLogger) => new EventHubQueueCache(checkpointer, bufferPool, timePurge, cacheLogger, this.SerializationManager);
+                CacheFactory = (partition, checkpointer, cacheLogger) =>
+                {
+                    var cache = new EventHubQueueCache(checkpointer, bufferPool, timePurge, cacheLogger, this.SerializationManager);
```

Comment:
nit:  We generally try to keep lambdas small, when they get large, we tend to put them into their own call.  This may be better as a CreateCacheFactory(..) function.